tests/macros/test_macros.py

```diff
@@ -89,3 +89,19 @@ def test_ds_format(ds, input_format, output_format, expected):
 def test_datetime_diff_for_humans(dt, since, expected):
     result = macros.datetime_diff_for_humans(dt, since)
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    "ds, expected",
```

Comment:
`ds` refers to date and known **ds** macro which might confuse devs. `value` or `input_value` more suitable here